TestCases/incomp_navierstokes/streamwise_periodic/README.md

```diff
@@ -0,0 +1,29 @@
+# Streamwise Periodicity testcases
+
+This folder contains the additional Testcases for streamwise periodic flow.
+A Tutorial can be found on the SU2 website.
+For all Testcases a gmsh .geo file is provided which allows to recreate/modify the mesh.
```

Comment:
😊 Thanks. I think I did myself a favour there. Especially now when addressing your comments I would build && runMyTestcases (which i made a separate file for) such that I directly knew if I got it right and because of the short time intervals it was clear what and where I broke s.th.. Feels a lot better to work like that when one has a soft&cozy safety net. I guess the next evolution step is TDD